tests/plugins/grpc/proto/health/Makefile

```diff
@@ -0,0 +1,16 @@
+PHP_OUT_DIR = ../../../../src
```

Comment:
Do not use some relative path here. Use directories in the grpc tests directory.

Reply:
you already keep the grpc service files for worker-grpc in this directory https://github.com/spiral/roadrunner-plugins/tree/master/tests/src

Reply:
Ok, I'll think about how to generate them on the fly later, now it's ok to use this directory. I'm still don't like this huge relative path, but ok, I'll think about how to optimize that by myself later.